### PR TITLE
fix: pass the language option for codeblock tag

### PR DESCRIPTION
**Pull request by xihale**
## What does it do?

I will pass the language option when I use either of `highlight.js` or `prism.js` for codeblock tag!

## Pull request tasks

- [x] Add test cases for the changes.
- [x] Passed the CI test.

### CODE REVIEW

**Review by SukkaW — CHANGES_REQUESTED** [review]
Would you mind adding unit test cases toward your changes?

**Comment by xihale** [comment]
> Would you mind adding unit test cases toward your changes?

but, I don't know what unit test I should add, can you help me?

**Comment by xihale** [comment]
> Would you mind adding unit test cases toward your changes?

but, I don't know what unit test I should add, can you help me?

**Comment by xihale** [comment]
I added a unit test to test my changes and find a inconspicuous bug then fixed it.

File: lib/plugins/tag/code.js [hunk 1]
```diff
@@ -118,9 +118,11 @@ function parseArgs(args) {
 
 module.exports = ctx => function codeTag(args, content) {
 
+  const options = parseArgs(args);
+
   // If neither highlight.js nor prism.js is enabled, return escaped code directly
   if (!ctx.extend.highlight.query(ctx.config.syntax_highlighter)) {
-    return `<pre><code>${escapeHTML(content)}</code></pre>`;
+    return `<pre><code${options.lang !== undefined ? ` class="language-${options.lang}"` : ''}>${escapeHTML(content)}</code></pre>`;
```
Comment (StephenNeelDurai) [inline lib/plugins/tag/code.js:125]: **Identical blocks of code found** on this line (125 and 140). I think you can create this line as separate variable and use it both places.

**Review by StephenNeelDurai — APPROVED** [review]

**Pull request closed without merging**